esmvaltool/diag_scripts/cvdp/cvdp_wrapper.py

```diff
@@ -127,11 +134,180 @@ def _nco_available():
     return ret
 
 
+def _is_png(path):
+    return os.path.basename(path).split('.')[-1] == 'png'
```

Comment:
It would be a bit nicer to use something like `Path(path).suffix == '.png'`, you can import `Path` from the `pathlib` standard library.

Reply:
Or simply `path.endswith('.png')`